web/src/components/nav/bar.js

```diff
@@ -1,7 +1,29 @@
 import { Link, NavLink, withRouter } from "react-router-dom";
 import { Container, Menu } from "../ui";
+import { logout } from "../../../src/components/auth/utils";
+
+let token = localStorage.getItem("access");
```

Comment:
This may be better served by making it part of the state of the navbar component. When we initialize the state, we can grab the initialize value of the token.